src/templates/forum/board_base.html

```diff
@@ -0,0 +1,82 @@
+{% extends "base.html" %}
```

Comment:
Can we keep the original extends statement from the django-machina template and modify the variable in django-settings? That way it remains consistent with the django-machina docs. I've tested it and it doesn't seem to cause any issues to do it this way.
```
{% extends MACHINA_BASE_TEMPLATE_NAME %}
```
Then put this at the end of settings/base.py. I've also included the setting to change the page title.
```
# DJANGO-MACHINA SETTINGS
# ------------------------------------------------------------------------------
# https://django-machina.readthedocs.io/en/stable/settings.html
MACHINA_FORUM_NAME = "chigame Forums"
MACHINA_BASE_TEMPLATE_NAME = "base.html"
```